z2pack/fp/_first_principles.py

```diff
@@ -162,7 +186,36 @@ def get_mmn(self, kpt):
                         format(i, overlaps.shape, shape)
                     )
 
-        return overlap_matrices
+        dmn_mat = None
+        if use_symm:
+            # if use_symm = True, we need to parse the .dmn file and write the result into symm_projectors
+            dmn_mat = dmn.get_dmn(self._dmn_path)
+        return overlap_matrices, dmn_mat
+
+    def suggest_symmetry_surfaces(self):
```

Comment:
As far as I can tell, this code is entirely independent of the other functionality of the ``System`` class -- so maybe it would be best to instead implement it as a separate helper function or class? This would avoid the problem that ``scf_folder`` and ``xml_path`` are specific to QE, since there could be a separate helper for other codes.